source/production/F0.Minesweeper.Logic.Abstractions/MinefieldFirstUncoverBehaviour.cs

```diff
@@ -0,0 +1,10 @@
+namespace F0.Minesweeper.Logic.Abstractions
+{
+	public enum MinefieldFirstUncoverBehaviour
```

Comment:
I prefer American English: "Behavior"
to be in sync with the other names, such as `DebugVisualization`

Reply:
Ooops, nice find 😉 
Done.